middleware/grpc_stats_test.go

```diff
@@ -211,7 +211,7 @@ func TestGrpcStatsStreaming(t *testing.T) {
 			received_payload_bytes_bucket{method="gRPC",route="/middleware.EchoServer/Process",le="1.048576e+08"} 5
 			received_payload_bytes_bucket{method="gRPC",route="/middleware.EchoServer/Process",le="2.62144e+08"} 5
 			received_payload_bytes_bucket{method="gRPC",route="/middleware.EchoServer/Process",le="+Inf"} 5
-			received_payload_bytes_sum{method="gRPC",route="/middleware.EchoServer/Process"} 1.5728664e+07
+			received_payload_bytes_sum{method="gRPC",route="/middleware.EchoServer/Process"} 1.5728689e+07
```

Comment:
Noting this difference suggests to me the code is seeing 5 bytes more on the wire.

Reply:
This turned out to be an artifact from this fix in grpc: https://github.com/grpc/grpc-go/pull/3886 , the old code was reporting wrong wirelength to the statistics callback actually.